.github/actions/setup-tag-env/action.yml

```
@@ -8,7 +8,10 @@ runs:
         ref_name="${{ github.ref_name }}"
         TAG="${{ github.sha }}"
         prefix="release/"
-        if [[ "$ref_name" == release/v* ]]; then
+        if [[ "$ref_name" == "main" ]]; then
+          TAG=v2.9-${{ github.sha }}-main 
```

Shouldn't it be v2.10?

```suggestion
          TAG=v2.10-${{ github.sha }}-main 
```